packages/react-ui-core/src/Svg/Svg.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export default class Svg extends Component {
+
+  static propTypes = {
+    fill: PropTypes.string,
+    height: PropTypes.number,
+    pathD: PropTypes.string.isRequired,
+    viewBox: PropTypes.string,
+    onClick: PropTypes.func,
+    size: PropTypes.number,
+    style: PropTypes.oneOfType([
+      PropTypes.array,
+      PropTypes.object,
+    ]),
+    width: PropTypes.number,
+    className: PropTypes.string,
+  }
+
+  render() {
+    const { pathD } = this.props
+    return (
+      <svg {...this.props} >
+        <path d={pathD} />
+      </svg>
+    )
```

Comment:
@lfender6445 has there been any discussion on this way of handing svg icons?

To me, this doesn't seem to be of any benefit. You may as well just write `<svg ...>` directly. It also limits us to just using `path`, and a single one at that. Take [this svg icon](https://raw.githubusercontent.com/rentpath/ag.js/master/src/apps/shared/Icon/icons/arrow_down.svg?token=AAOby1kduMYgJDWSVYOf6VsvxR5MfcpLks5ZghDHwA%3D%3D) for example. Or [this one](https://raw.githubusercontent.com/rentpath/ag.js/master/src/apps/shared/Icon/icons/crosshair.svg?token=AAOby9g1HEo6y1tI-jwtKDAoFJu4Ceobks5ZghFzwA%3D%3D).

Maintaining a bunch of path strings in JS seems like a maintenance headache and a step back from what AG.js does, with SVG sprites and `xlink:href`